include/mgba/internal/gba/renderers/gl.h

```diff
@@ -16,7 +16,9 @@ CXX_GUARD_START
 #include <mgba/internal/gba/renderers/common.h>
 #include <mgba/internal/gba/video.h>
 
-#if defined(BUILD_GLES2) || defined(BUILD_GLES3)
```

Comment:
This is definitely not correct--`BUILD_GL` corresponds to OpenGL 1.x or OpenGL ES 1.x, but this requires OpenGL 3.x or OpenGL ES 3.x.